### DISCUSSION TITLE
Project Fully Resuming Soon

### DISCUSSION THREAD

**Discussion by Mrbt0907** [CosmicHikari/Weather2-Remastered#20] [Announcements]
I apologize for the lack of updates with the project. I was slowly updating the mod, but I wasn't making any meaningful progress since my attention was on another project. I'd say in about a weeks time, I should have a lot more time to work on this mod.

My new goal is to get the optimizations complete and get an alpha out to see what needs to be done. Features should be done by the time I have another test build out. I'll let you know when I make significant progress with the update. See you guys then.

**Comment by Justin7654** [comment]
:)
